### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by Adams-Ghub**
- Developed the navbar for desktop
- Developed the headline section for desktop
- Developed the Project section for desktop
- Developed the About section for desktop
- Developed the Contact section for desktop
- Developed the footer section for desktop

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,229 +1,344 @@
...
+        alt="geometry-image-seven"
+        id="geometry-image-seven"
+      />
+    </section>
+    <footer class="footer-section">
+      <a class="get-resume-btn" href="#">
+        Get my Resume <img src="./images/ic_link_teal.png" alt="" />
+      </a>
+      <div class="footer-social-links">
+        <a href="#"><img src="./images/ic_stackoverflow.png" /></a>
```
Comment (Kossi-stack) [inline index.html]: - [ ] Kindly use a ul tag to group them, and li per icon; each icon has a link (a tag).

- Apart from being semantically correct, the a tag will let you connect your social networks with the icons, using href.
- [ ] And also fix them, they are not showing.
- Figma
Screenshot 2023-03-03 at 03 06 06

- Your Design
Screenshot 2023-03-03 at 03 07 23

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @Adams-Ghub,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comment under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 2]
```diff
@@ -1,229 +1,360 @@
...
+        </li>
+        <li>
+          <a href="#"><img src="./images/linkedin.png" alt="linkedin-img" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="./images/ic_medium.png" alt="medium-img" /></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa fa-angellist" id="icon-footer"></i></a>
+        </li>
```
Comment (Felix45) [inline index.html:356]: - [ Optional ] Please consider adding links to your social media pages such as `GitHub` `Twitter` `LinkedIn` `Medium` and `Angelist` these links will help other online users such as recruiters to connect with you online. Currently, your `href` attributes are empty which means the links don't lead to your social media pages.

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @Adams-Ghub , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] No linter errors :white_check_mark: 
- [x] Good descriptive commit messages :white_check_mark: 
- [x] GitHub flow is correctly used :white_check_mark:  
- [x] Implemented all the changes for the previous reviewer :white_check_mark:  
- [x]  Descriptive pull request title and summary :white_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Adams-Ghub) [reply index.html:356]: Thank you so much. I will do that.

**Pull request merged**